docs/integrations/builtin/credentials/actionnetwork.md

```diff
@@ -12,19 +12,22 @@ You can use these credentials to authenticate the following nodes:
 
 ## Prerequisites
 
-- Create an [Action Network](https://actionnetwork.org/) account.
-- [Contact](https://actionnetwork.org/contact) Action Network to become a partner and enable API access.
+- Create an [Action Network](https://actionnetwork.org/){:target=_blank .external-link} account.
+- [Contact](https://actionnetwork.org/contact){:target=_blank .external-link} Action Network to become a partner and enable API access.
+- If you are a developer needing API Access for development work, you can also [Request a developer account](https://actionnetwork.org/developers){:target=_blank .external-link} to enable partner access.
 
-## Using API key
+## Supported authentication methods
 
-After becoming an Action Network partner:
+- API Key
 
-1. Open your Action Network dashboard.
-2. From the **Start Organizing** menu navigate to **Details** > **API & Sync**.
-3. Use the dropdown to select the list you want to generate a key for, your personal list or that of any groups you are an administrator of.
-4. From the **Your API Key** section, generate a new API key:
+## Related resources
 
-![API & Sync Page](/_images/integrations/builtin/credentials/actionnetwork/action_network_api_key.png)
+Refer to [Action Network's API documentation](https://actionnetwork.org/docs/){:target=_blank .external-link} for more information about working with the service.
 
-5. Copy your new key and use it with your Action Network node credentials in n8n.
+## Using API Key
 
+To configure this credential, you'll need:
+
+- An **API Key**
```

Comment:
```suggestion
- An API key
```

Reply:
Bold is usually only for UI fields. I'm guessing the node has a field named API Key? But this isn't really a reference to the field.